From: Michael Olbrich <m.olbrich@pengutronix.de>
Date: Wed, 18 May 2016 10:16:15 +0200
Subject: [PATCH] sleep-monitor-upower: add missing include

Otherwise building fails with:

nm-sleep-monitor-upower.c:132:45: error: expected declaration specifiers or '...' before 'nm_sleep_monitor_get'
nm-sleep-monitor-upower.c:132:67: error: expected declaration specifiers or '...' before '(' token

Signed-off-by: Michael Olbrich <m.olbrich@pengutronix.de>
---
 src/nm-sleep-monitor-upower.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/nm-sleep-monitor-upower.c b/src/nm-sleep-monitor-upower.c
index 5f0ff24552b2..8d0719039ccf 100644
--- a/src/nm-sleep-monitor-upower.c
+++ b/src/nm-sleep-monitor-upower.c
@@ -24,6 +24,7 @@
 #include <sys/stat.h>
 
 #include "nm-core-internal.h"
+#include "NetworkManagerUtils.h"
 
 #include "nm-sleep-monitor.h"
 
